src/lua/fiber.c

```diff
@@ -138,14 +138,24 @@ lbox_pushfiber(struct lua_State *L, struct fiber *f)
 	lua_rawgeti(L, LUA_REGISTRYINDEX, fid_ref);
 }
 
+/**
+ * Returns fiber id for the fiber located at the given `index` on the Lua stack.
+ */
+static uint64_t
+luaT_fiber_id(struct lua_State *L, int index)
```

Comment:
Nit: There are no other functions using `luaT_` prefix in this file. Let's change to `lbox_checkfiberid` for consistency.

Reply:
> It is not strictly followed across the code, but AFAIR there is a convention that `lbox_` prefixed functions adhere the [lua_CFunction](https://www.lua.org/manual/5.1/manual.html#lua_CFunction) prototype (so they can be passed to [lua_pushcfunction](https://www.lua.org/manual/5.1/manual.html#lua_pushcfunction) and exposed to the Lua land). Other functions that need the Lua state are usually prefixed with `luaT_` and they can accept more than just `struct lua_State *`.

- https://github.com/tarantool/tarantool/pull/8356#discussion_r1111337624

Reply:
I remember about that one but this just looks inconsistent with the existing code in this file.

Reply:
Renamed.